autoinstall/Pardus/preseed.cfg

```diff
@@ -0,0 +1,250 @@
+d-i apt-setup/contrib boolean true
+
+# Disable upgrading while installation
+d-i pkgsel/upgrade select none
+
+### Package selection
+#tasksel tasksel/first multiselect standard, web-server, kde-desktop
+tasksel tasksel/first multiselect standard, desktop
+tasksel tasksel/desktop multiselect xfce
+
+# Individual additional packages to install
+# There is no open-vm-tools-desktop openssh-server and cloud-init in CDROM
+#d-i pkgsel/include string build-essential open-vm-tools openssh-server sg3-utils vim
+
+# Policy for applying updates. May be "none" (no automatic updates),
+# "unattended-upgrades" (install security updates automatically), or
+# "landscape" (manage system with Landscape).
+d-i pkgsel/update-policy select none
+
+# Some versions of the installer can report back on what software you have
+# installed, and what software you use. The default is not to report back,
+# but sending reports helps the project determine what software is most
+# popular and should be included on the first CD/DVD.
+popularity-contest popularity-contest/participate boolean false
+
+### Boot loader installation
+# Grub is the boot loader (for x86).
+
+# This is fairly safe to set, it makes grub install automatically to the UEFI
+# partition/boot record if no other operating system is detected on the machine.
+d-i grub-installer/only_debian boolean true
+
+# Due notably to potential USB sticks, the location of the primary drive can
+# not be determined safely in general, so this needs to be specified:
+#d-i grub-installer/bootdev string /dev/sda
+# To install to the primary device (assuming it is not a USB stick):
+d-i grub-installer/bootdev string default
+
+# Avoid that last message about the install being complete.
+d-i finish-install/reboot_in_progress note
+
+# This will prevent the installer from ejecting the CD during the reboot,
+# which is useful in some situations.
+#d-i cdrom-detect/eject boolean true
+
+### Preseeding other packages
+# Depending on what software you choose to install, or if things go wrong
+# during the installation process, it's possible that other questions may
+# be asked. You can preseed those too, of course. To get a list of every
+# possible question that could be asked during an install, do an
+# installation, and then run these commands:
+   debconf-get-selections --installer > /target/root/preceed.cfg
+   debconf-get-selections >> /target/root/preceed.cfg
+
+# This command is run just before the install finishes, but when there is
+# still a usable /target directory. You can chroot to /target and use it
+# directly, or use the apt-install and in-target commands to easily install
+# packages and run commands in the target system.
+d-i preseed/late_command string \
+    echo "Check network ..." > /target/dev/ttyS0; \
+    ip addr show > /target/dev/ttyS0; \
+    echo "Linkup network ..." > /target/dev/ttyS0; \
+    chroot /target /sbin/ifconfig $(/sbin/ifconfig -a | grep '^[a-z]' | cut -d: -f1 | head -n 1) up; \
```

Comment:
Why can't the network interface be up automatically?

Reply:
network interface is linkdown during autoinstall. I try to linkup it with d-i  but failed.  Not sure why.
So i have to linkup the nic in d-i preseed/late_command


Best regards
Yuhua Zou